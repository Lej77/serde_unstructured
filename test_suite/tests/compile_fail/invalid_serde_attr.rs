
#[derive(serde_unstructured::SerdeView)]
struct Data {
    #[serde(what)]
    field: u8,
}

fn main() {}