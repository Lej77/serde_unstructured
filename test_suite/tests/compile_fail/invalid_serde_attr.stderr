error: `SerdeView` failed at `serde` attribute: unknown serde field attribute `what`

         = help: skip a field using a `#[serde_view(skip)]` attribute or use an empty `#[serde_view()]` attribute to ignore any `#[serde]` attributes.

 --> tests/compile_fail/invalid_serde_attr.rs:4:13
  |
4 |     #[serde(what)]
  |             ^^^^
